snake.cpp

```diff
@@ -0,0 +1,92 @@
+#include"bits/stdc++.h"
```

Comment:
Please create classes

Reply:
added

Reply:
please create multiple classes for each entity

Reply:
give it a try

Reply:
multiple classed added